zds/tutorialv2/tests/tests_views/tests_content.py

```diff
@@ -6278,3 +6279,22 @@ def test_add_help_opinion(self):
                                     'help_wanted': help_wanted.title})
         self.assertEqual(400, resp.status_code)
         self.assertEqual(0, PublishableContent.objects.filter(pk=tutorial.pk).first().helps.count())
+
+    def test_save_no_redirect(self):
+        self.client.login(username=self.user_author.username, password='hostel77')
+        tutorial = PublishableContentFactory(author_list=[self.user_author])
+        extract = ExtractFactory(db_object=tutorial, container=tutorial.load_version())
+        tutorial = PublishableContent.objects.get(pk=tutorial.pk)
+        resp = self.client.post(reverse('content:edit-extract', args=[tutorial.pk, tutorial.slug, extract.slug]),
+                                {
+                                    'last_hash': extract.compute_hash(),
+                                    'text': 'a brand new text',
+                                    'title': extract.title,
+                                    'msg_commit': 'a commit message'},
+                                HTTP_X_REQUESTED_WITH='XMLHttpRequest', follow=False)
+        # no redirect
+        self.assertEqual(200, resp.status_code)
+        result = loads(resp.content.decode('utf-8'))
+        self.assertEqual('ok', result.get('result', None))
+        tutorial = PublishableContent.objects.find(pk=tutorial.pk)
```

Comment:
Petite erreur que je n'avais pas vue et qui fait planter les tests ^^
```suggestion
        tutorial = PublishableContent.objects.get(pk=tutorial.pk)
```

Reply:
c'est étrange ça, c'est sensé être fixé depuis un baille